build/litsearch/getInternalData.py

```diff
@@ -267,6 +282,12 @@ def mergePaperDataFrames(dataFramesList):
         dataFramesList_DOI.append(df[validDOI])
         dataFramesList_NoDOI.append(df[invalidDOI])
 
+    # Check if there are issues with duplicate DOIs
+    for df in dataFramesList_DOI:
+        duplicateDOIs = getDuplicates(list(df.index))
+        if len(duplicateDOIs) > 0:
+            raise ValueError('The following paper(s) has/have duplicate issues:', duplicateDOIs)
```

Comment:
I agree that we want the build to fail if that are duplicates.  That will be a stronger signal to maintainers to close the duplicate issues.